packages/story-editor/src/components/panels/design/textStyle/fontPicker.js

```diff
@@ -187,6 +187,7 @@ const FontPicker = forwardRef(function FontPicker(
   return (
     <Datalist.DropDown
       ref={ref}
+      zIndex={10}
```

Comment:
This will change the z-index for all instances of the font picker.

It would be better for the FontCheckDialog to pass this as a prop.